Content.Server/DeviceLinking/Systems/SignalSwitchSystem.cs

```diff
@@ -38,8 +38,15 @@ private void OnActivated(EntityUid uid, SignalSwitchComponent comp, ActivateInWo
             _deviceLink.SendSignal(uid, comp.StatusPort, comp.State);
         }
 
+        var ev = new SwitchPressedEvent();
+        RaiseLocalEvent(uid, ref ev);
+
         _audio.PlayPvs(comp.ClickSound, uid, AudioParams.Default.WithVariation(0.125f).WithVolume(8f));
 
         args.Handled = true;
     }
 }
+
+[ByRefEvent]
+// Event to inform any other systems that may want to know that the signal switch was pressed
```

Comment:
Use [XMLDoc](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/), not the regular comments